server/datastore/mysql/migrations/tables/20230814150442_AddHostScriptResultsTable.go

```diff
@@ -0,0 +1,67 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230814150442, Down_20230814150442)
+}
+
+func Up_20230814150442(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE host_script_results (
```

Comment:
I really like this idea of adding some comments to the migration with the table definition. Thank you

Reply:
Seconded